### PR TITLE
Configable taggables morph table

### PR DESCRIPTION
**Pull request by silnex**
The tag class can be customized with `getTagClassName` method, 
but what if we add the ability to set the `taggables` table?

### CODE REVIEW

File: src/HasTags.php [hunk 1]
```diff
@@ -19,6 +19,16 @@ public static function getTagClassName(): string
         return config('tags.tag_model', Tag::class);
     }
 
+    public static function getTaggableMorphName(): string
```
Comment (freekmurze) [inline src/HasTags.php:22]: Make these non-static

Comment (silnex) [reply src/HasTags.php:22]: Should I just make `getTaggableMorphName` non-static?
or with `getTaggableTableName` ?

Comment (freekmurze) [reply src/HasTags.php]: Let's make both non-static 👍

Comment (silnex) [reply src/HasTags.php]: done!

**Comment by freekmurze** [comment]
Thanks!

**Pull request merged**